frontend/src/pages/InducteeSignUpPage/index.tsx

```diff
@@ -47,27 +47,25 @@ function InducteeSignUpPage(): JSX.Element {
     try {
       await createNewInducteeUser(signupSubmission);
     } catch {
-      console.log('Create new user failed');
+      alert('Create new user failed');
       setSubmitting(false);
       return;
     }
 
     try {
       await doSignInWithEmailAndPassword(email, password, false);
     } catch {
-      console.log('Sign in failed');
+      alert('Sign in failed');
       setSubmitting(false);
       return;
     }
 
     try {
       await doSendVerificationEmail();
     } catch {
-      console.log('Send verification email failed.');
+      alert('Send verification email failed.');
     }
     await doSignOut();
-    setSubmitting(false);
-    alert('You have successfully signed up for an account.');
```

Comment:
Why are we taking out setSubmitting and the alert afterwards? Shouldn't they be useful to determine that account creation was successful?

Reply:
Isn't it taken out because the verification failed following the alert on line 66?